### ISSUE TITLE
Impossible to enter the main navigation with keyboard

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#83]
The main menu in the header of MDN Web Docs is not accessible when using a keyboard. While a user can tab to the top level items, there is no way to trigger the drop down and tab into the various entries.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563467

**Comment by schalkneethling** [comment]
Also see https://bugzilla.mozilla.org/show_bug.cgi?id=1564836
